cookbooks/wiki/templates/default/mw-ext-Translate.inc.php.erb

```diff
@@ -0,0 +1,36 @@
+<?php
+# DO NOT EDIT - This file is being maintained by Chef
+
+wfLoadExtension( 'Translate' );
+$wgGroupPermissions['user']['translate'] = true;
+$wgGroupPermissions['user']['skipcaptcha'] = true; // Bug 34182: needed with ConfirmEdit
```

Comment:
Bugzilla 34182 is https://phabricator.wikimedia.org/T36182

Reply:
Though the bug, I am not sure if it is good to disable the captcha for all logged-in users.
In another perspective, does OSM wiki have ConfirmEdit enabled? (Is this line needed?)
Anyway, I think it would be good if 'autoconfirmed' is used than 'user'.